garrysmod/lua/entities/widget_axis.lua

```diff
@@ -115,6 +122,24 @@ function ENT:Setup( ent, boneid, rotate )
 
 end
 
+--
+-- This was added by Seafoury
+-- widget_axis.lua
+--
+-- if ENT:SetXYZ() and ENT:SetAxisIndex() don't work
+-- then please tell me and I will try to fix it! (:
+function ENT:SetXYZ( x, y, z, d )
+	base.ArrowZ:SetColor( Color( x, y, z, d ) )

```

Comment:
What is this meant to do/achieve?
Why is it only applied to `ArrowZ`?
Again, `base` in this context is `nil`, so this would just error?